src/test/java/org/broadinstitute/consent/http/db/DatasetDAOTest.java

```diff
@@ -524,29 +527,31 @@ public void testCreateNumberTypedDatasetProperty() {
     @Test
     public void testCreateDateTypedDatasetProperty() {
         Dataset d = insertDataset();
-        Instant now = Instant.now();
+        LocalDate date = LocalDate.now();
```

Comment:
So, here the user is inputting a simple date. e.g., the alternative data sharing plan release date is `2022-10-11`. That's all the information we have. A LocalDate makes sense because it makes no assumptions other than keeping the year, day, and month, which is all we want to store. To make an Instant we would have to add information which we don't have, unless we changed what the frontend should send. I think an instant does make sense for most use cases. But, I think it makes sense here, because all we need to know is the day of the year it happened; we don't need to know timezone really. I tried keeping the Instant at first, but it became somewhat complicated having to insert timezone information, and I ended up just giving it the system's timezone, which isn't really any useful information. After spending more than like 20 minutes trying to get all the date conversions and format things right and it not working, I figured it was simpler. I am happy  convert it back to an Instant if you all prefer, but it takes some non-trivial code to add timezone information.

Reply:
One question is: does the service need to keep a globally consistent store of time and dates in the system? If so, then I think it's worth thinking about using something like an `instant`. If you say your sharing plan release date is 2022-10-11, is this in relation to a particular geography (say, where the NIH is headquartered) or something else? I agree with Nick that it's worth having a larger conversation around this.

Reply:
I talked to Jonathan, and I don't think it's relative to anything. He said it would be ok to default to UTC. If it's preferred, I could convert these to an Instant with a UTC timezone, but there were some weird parsing issues when I did so before. I think I should be able to figure it out with some work though.

Reply:
I think typically it's better to default to UTC and wait until the data gets to the client to render the date but it's your call.  My main concern here is if we are recording dates based on the user's locale but not recording the timezone, because we then don't actually know the time.  Is this getting stored in the DB?